src/components/RefinementList/RefinementList.js

```diff
@@ -127,6 +128,8 @@ class RefinementList extends React.Component {
     }
 
     const limit = this.state.isShowMoreOpen ? this.props.limitMax : this.props.limitMin;
+    this.props.facetValues = this.props.facetValues.slice(0, limit);
```

Comment:
What do you mean by not doable? The above code is currently working. 

Is that a React best practice?